app/editor/src/store/slices/app/useAppStore.ts

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useAppDispatch, useAppSelector } from 'store';
+import { useDeepCompareMemo } from 'tno-core';
+
+import { addRequest, clearRequests, removeRequest } from '.';
+import { IAppState } from './interfaces';
+
+export interface IAppStore {
+  addRequest: (url: string) => void;
+  removeRequest: (url: string) => void;
+  clearRequests: () => void;
+  isLoading: () => boolean;
+}
+
+export const useAppStore = (): [IAppState, IAppStore] => {
```

Comment:
This hook is a wrapper to handle all redux stuff.  We don't want `dispatch` or `useAppSelector` used anywhere elese in the application!